apps/heft/src/startWithVersionSelector.ts

```diff
@@ -89,9 +89,9 @@ function tryStartLocalHeft(): boolean {
 
       // To avoid a loading the "resolve" NPM package, let's assume that the Heft dependency must be
```

Comment:
Both the CJS and the ESM resolvers that Node uses can be invoked externally without npm packages. `require.resolve` is sufficient here.